lib/imagec/download.go

```diff
@@ -335,7 +341,7 @@ func (ldm *LayerDownloader) makeDownloadFunc(layer *ImageWithMeta, ic *ImageC, p
 
 // makeDownloadFuncFromDownload returns a func used by the TransferManager to
 // handle a layer that was already seen in this image pull, or is currently being downloaded
-func (ldm *LayerDownloader) makeDownloadFuncFromDownload(layer *ImageWithMeta, sourceDownload, parentDownload *downloadTransfer, layers []*ImageWithMeta) xfer.DoFunc {
+func (ldm *LayerDownloader) makeDownloadFuncFromDownload(op trace.Operation, layer *ImageWithMeta, sourceDownload, parentDownload *downloadTransfer, layers []*ImageWithMeta) xfer.DoFunc {
 
 	return func(progressChan chan<- progress.Progress, start <-chan struct{}, inactive chan<- struct{}) xfer.Transfer {
```

Comment:
It looks like this function may need exactly the same change as made for `makeDownloadFunc` above.

Reply:
It may need it in the future, but right now "op" is not used inside the body of the returned function